Sources/CorePayments/Networking/AnalyticsService.swift

```diff
@@ -2,8 +2,8 @@ import Foundation
 
 /// Constructs `AnalyticsEventData` models and sends FPTI analytics events.
 @_documentation(visibility: private)
-public struct AnalyticsService {
-    
+public struct AnalyticsService: @unchecked Sendable {
```

Comment:
Curious about the reason behind using @unchecked here.

Reply:
With @unchcked, the Sendable compiler warnings are silenced, assuring that type is safe to pass between concurrent contexts. Without it, I get a warning on Sendable capture in sendEvent function waiting performEventRequest. This is because the Task block creates an asynchronous context. Since we are just sending data to analytics endpoint, I don't think there is danger of data races. The values of coreConfig and orderID are not changing after initialization.

Reply:
What do you think about making `CoreConfig` and `TrackEventsAPI` conform to `Sendable`? Would this be more maintainable over time since any new updates could introduce something that is unsafe?